### PR TITLE
feat(ex/skate/ors): format directions as "short directions"

### PR DESCRIPTION
**Pull request by firestack**
Asana Ticket: https://app.asana.com/0/1208502791134829/1208288732617333/f

### CODE REVIEW

File: test/skate/open_route_service_api_test.exs [hunk 1]
```diff
@@ -84,15 +84,18 @@ defmodule Skate.OpenRouteServiceAPITest do
                "steps" => [
                  %{
                    "instruction" => "not-in-results",
+                   "name" => "not-in-results",
                    # Depart
                    "type" => 11
                  },
                  %{
                    "instruction" => "1",
+                   "name" => "1",
```
Comment (hannahpurcell) [inline test/skate/open_route_service_api_test.exs:93]: non-blocking: Could the names added here look more like street names? I was confused to see the instruction and name be the same value and had to go look up the difference between the two. Seems like `name` should be the street name?

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    do: "L - #{name}"
+
+  defp format_instruction(:right, name, _),
+    do: "R - #{name}"
+
+  defp format_instruction(:slight_left, name, _),
+    do: "L - #{name}"
+
+  defp format_instruction(:slight_right, name, _),
+    do: "R - #{name}"
```
Comment (hannahpurcell) [inline lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex:39]: fix: [Looks like](https://app.asana.com/0/0/1208288732617333/1208592058043925/f) "Slight Left / Right" should be "Slight L / R"

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  defp format_instruction(:keep_left, name, _),
+    do: "L - #{name}"
+
+  defp format_instruction(:keep_right, name, _),
+    do: "R - #{name}"
```
Comment (hannahpurcell) [inline lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex:45]: fix: [Looks like](https://app.asana.com/0/0/1208288732617333/1208592058043925/f) "Keep Left / Right" should be "Keep L / R"

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+  defp format_instruction(:sharp_left, name, _),
+    do: "Sharp L - #{name}"
+
+  defp format_instruction(:sharp_right, name, _),
+    do: "Sharp R - #{name}"
```
Comment (hannahpurcell) [inline lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex:51]: fix: [Looks like](https://app.asana.com/0/1208502791134829/1208288732617333/f) "Sharp L / R" should just be "L / R"

**Review by hannahpurcell — COMMENTED** [review]
The simplified instructions for keep / sharp / slight are implemented differently from the original ticket. Was there follow-up discussion not captured in the ticket that I'm missing?

File: test/skate/open_route_service_api_test.exs (hunk 1 above)
Comment (firestack) [reply test/skate/open_route_service_api_test.exs]: https://github.com/mbta/skate/compare/ae88499aacd89a6816c12859051a1afae3b29e4e..c588d63d5bd2ffa8d4ba61c0d4c68f933552519f

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex (hunk 2 above)
Comment (firestack) [reply lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex]: https://github.com/mbta/skate/compare/c588d63d5bd2ffa8d4ba61c0d4c68f933552519f..c5c07d6c55b1b10a35234a6fe601a553961ace52

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex (hunk 3 above)
Comment (firestack) [reply lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex]: https://github.com/mbta/skate/compare/c588d63d5bd2ffa8d4ba61c0d4c68f933552519f..c5c07d6c55b1b10a35234a6fe601a553961ace52

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex (hunk 4 above)
Comment (firestack) [reply lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex:51]: https://github.com/mbta/skate/compare/c588d63d5bd2ffa8d4ba61c0d4c68f933552519f..c5c07d6c55b1b10a35234a6fe601a553961ace52

**Comment by firestack** [comment]
Feel free to take a look if you have time @bfauble!

File: lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex (hunk 4 above)
Comment (firestack) [reply lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex]: Whoops wrong thing 
image

**Review by hannahpurcell — APPROVED** [review]
Great, thanks! This will be so nice to have!

**Comment by firestack** [comment]
Whoops I forgot automerge was on, if you were going to or were reviewing it, sorry that got swept out from under you @bfauble. I was gonna give you a bit of time.

**Pull request merged**